dune-project

```diff
@@ -1,3 +1,3 @@
-(lang dune 1.9)
-(using coq 0.1)
-(name coq-bignums)
+(lang dune 2.5)
+(using coq 0.2)
+(name bignums)
```

Comment:
@palmskog is it normal that the coq-community template enforces this change?
(I'm not very familiar with dune but I'd expact the `(name _)` value should be the same as dune's `(package _)` value…)

Reply:
The problem is that many projects that use dune will have not only a package `coq-<name>`, but also other packages with different names. So having `name` be exactly `coq-<name>` is misleading. My reasoning was that it makes sense to use just `<name>` in the `dune-project file since it will be joint for all packages. But if @ejgallego has reservations I can fix this in the templates.

Reply:
I've yet to understand what is the point of providing a `(name _)` in a `dune-project` file (and yet, it is compulsory).